### DISCUSSION TITLE
screenshots

### DISCUSSION THREAD

**Discussion by d0rfl** [factropy/community#4] [Show and tell]
Not the first main-bus ever in the game, but one that stuck around for testing for a long time early on :)

![screen1](https://img.itch.zone/aW1hZ2UvMTM5NDkwNS84MTI5MTExLmpwZw==/original/2k9XHW.jpg)

**Comment by freemin7** [comment]
![Bildschirmfoto vom 2022-08-30 13-58-29](https://user-images.githubusercontent.com/25376460/187656999-122c4721-d703-4bd8-b424-efe579299a5d.png)
